src/components/stats/garden/composter-upgrades.svelte

```diff
@@ -0,0 +1,124 @@
+<script lang="ts">
+	import {
+		getCropDisplayName,
+		getComposterUpgradeDisplayName,
+		getSpecialCropDisplayName,
+		Crop,
+	} from 'farming-weight';
+	import type { components } from '$lib/api/api';
+	import { PROPER_CROP_TO_IMG, SPECIAL_CROP_TO_IMG } from '$lib/constants/crops';
+	import { API_COMPOSTER_UPGRADE_TO_UPGRADE, COMPOSTER_UPGRADE_TO_IMG } from '$lib/constants/composter';
+	import * as Popover from '$ui/popover';
+	import { getComposterUpgradeCost, getEnchantedCropCollectionAmount } from '$lib/calc/garden';
+
+	interface Props {
+		garden?: components['schemas']['GardenDto'];
+	}
+
+	let { garden = undefined }: Props = $props();
```

Comment:
Please use `getStatsContext()` to get the garden data like done in other components instead of passing it in as a prop here. (I know some of the other garden components don't use it yet but they will soon)